### ISSUE TITLE
onxxx link not rewritten

### ISSUE THREAD

**Issue by rgaudin** [openzim/warc2zim#209]
Haven't investigated enough to understand which component is responsible but to reproduce:

- Go to https://dev.library.kiwix.org/viewer#solidarite-numerique_fr_2023-03/
- Click on any of the cards

You must be **using Chrome**.

This leads to the broken content Chrome response with the following message in the console:

```
Refused to frame 'https://www.solidarite-numerique.fr/' because it violates the following Content Security Policy directive: "default-src 'self' data: blob: mediastream: ws: wss:". Note that 'frame-src' was not explicitly set, so 'default-src' is used as a fallback.
```

What happens is that those _links_ are location change emitted in a _click_ event from an `onclick` attribute.

```html
<aside onclick="document.location.href='https://www.solidarite-numerique.fr/tutoriels/utiliser-le-decodex-pour-verifier-les-fausses-informations-ou-fake-news/?thematique=sinformer';" class="sinformer"></aside>
```

Not sure if Wombat should detect and fix this…

Firefox doesn't exhibit the same behavior if you're online as it will display the online content.

**Comment by mgautierfr** [comment]
> The problem is that none onxxx tags are rewritten, while they probably should. @mgautierfr WDYT?

I think I have missed that. We should. (And wabac is doing it : https://github.com/webrecorder/wabac.js/blob/main/src/rewrite/html.js#L143-L145)

**Comment by kelson42** [comment]
Fixed by #270
